src/Ryujinx.Ui.Common/App/ApplicationLibrary.cs

```diff
@@ -905,6 +905,11 @@ public static (Nca patch, Nca control) GetGameUpdateData(VirtualFileSystem fileS
                 {
                     updatePath = JsonHelper.DeserializeFromFile(titleUpdateMetadataPath, _titleSerializerContext.TitleUpdateMetadata).Selected;
 
+                    if (!string.IsNullOrWhiteSpace(updatePath) && !File.Exists(updatePath))
+                    {
+                        Logger.Warning?.Print(LogClass.Loader, $"Update file \"{updatePath}\" for titleId: {titleId} was not found! Game saves can become corrupted!");
```

Comment:
```suggestion
                        Logger.Warning?.Print(LogClass.Loader, $"Update file \"{updatePath}\" for ApplicationId: {titleId} was not found! Game saves can become corrupted!");
```

(The var have to be renamed but that can be done later)

Reply:
Fixed